internals/overlord/state/identities.go

```diff
@@ -18,8 +18,11 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"regexp"
 	"sort"
 	"strings"
+
+	"github.com/GehirnInc/crypt/sha512_crypt"
```

Comment:
I there any reason why we use this library and not ```crypto/sha512``` for hashing the password ?

Reply:
Yes, they're different things: `crypto/sha512` is the straight SHA-512 hash algorithm, but `GehirnInc/crypt/sha512_crypt` implements the [SHA-crypt](https://www.akkadia.org/drepper/SHA-crypt.txt) algorithm using SHA-512 hashes and a whole bunch of "rounds" and other stuff I don't understand :-) to make it good for use with passwords (slow and secure, basically).

`GehirnInc/crypt/sha512_crypt` [uses](https://github.com/GehirnInc/crypt/blob/8cc1b52080c5761aa65952b1c39dd889bf54d76c/sha512_crypt/sha512_crypt.go#L65) the stdlib's `crypt/sha512` in its implementation.

Reply:
Thanks! Makes sense.